README.md

```diff
@@ -4,3 +4,16 @@ Command to run all test-cases in folder
 
 ``` 
 pytest src/modules/example/test/
+
```

Comment:
@qaivanpleten please use proper fromatting. it looks not so good - https://github.com/qaivanpleten/sp_learning/blob/863e33ef11c291a9708b9bd47d8fd7be6850cf5d/README.md